cg/apps/scout/scoutapi.py

```diff
@@ -49,7 +49,9 @@ def upload(self, scout_load_config: Path, force: bool = False) -> None:
         self.process.run_command(load_command)
         LOG.debug("Case loaded successfully to Scout")
 
-    def export_panels(self, panels: list[str], build: str = GENOME_BUILD_37) -> list[str]:
+    def export_panels(
+        self, panels: list[str], build: str = GENOME_BUILD_37, dry_run: bool = False
```

Comment:
that was my intention at first but I chicken out since I saw it had quite a lot of usages and didn't want to make this PR bigger and make things fail. I could include it though

Reply:
I see... I would try to change it but it's up to you. When it's parsed as string the conversion is done automatically f"{GenePanelGenomeBuild. GENOME_BUILD_37}". For other cases I think we would need to add `.value` (to this one for example: (`["--build", build])`).

Reply:
I'm leaving this for now. Too many issues already in this PR

Reply:
Will address it in a dedicated PR

Reply:
Yes sounds good 👍